docs/release_notes/sat_2.5_release_notes.md

```diff
@@ -1,10 +1,10 @@
 # Changes in SAT 2.5
 
-The 2.5.17 version of the SAT product includes:
+The 2.5.21 version of the SAT product includes:
```

Comment:
I just created 2.5.22, actually. See PR here: https://github.com/Cray-HPE/sat-product-stream/pull/68/files

```suggestion
The 2.5.22 version of the SAT product includes:
```

Reply:
Updated in next commit.